ocaml-lsp-server/src/ocaml_lsp_server.ml

```diff
@@ -163,23 +167,45 @@ let on_initialize rpc (ip : Lsp.Types.InitializeParams.t) =
   let state = { state with init = Initialized ip.capabilities } in
   Fiber.return @@ Ok (initialize_info, state)
 
+let result_of_list l =
+  let rec aux acc = function
+    | [] -> Ok (acc |> List.rev)
+    | Error err :: _rest -> Error err
+    | Ok el :: rest -> aux (el :: acc) rest
+  in
+  aux [] l
```

Comment:
This looks like `Result.List.all` to me.

Reply:
Right, thanks, I'm not familiar with Stdune yet 🙂